app/src/main/java/com/x8bit/bitwarden/data/auth/repository/AuthRepositoryImpl.kt

```diff
@@ -385,6 +388,13 @@ class AuthRepositoryImpl(
         mutableHasPendingAccountDeletionStateFlow.value = false
     }
 
+    override suspend fun getShowWelcomeCarousel(): Boolean =
+        !settingsRepository.hasUserLoggedInOrCreatedAccount &&
+            featureFlagManager.getFeatureFlag(
+                key = FlagKey.OnboardingCarousel,
+                forceRefresh = true,
```

Comment:
Do we need this to refresh here?

We refresh the flag on launch already

Reply:
Good point, thank you!

Reply:
You may need this: https://github.com/bitwarden/android/pull/3692

Reply:
Ah that does make things a lot easier! Thank you!! 😍